### PR TITLE
feat: update lazy ape logo [WEB-488]

### PR DESCRIPTION
**Pull request by patcito**

### CODE REVIEW

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found address [0xd533a949740bb3306d119cc777fa900ba034cd52](https://etherscan.io/address/0xd533a949740bb3306d119cc777fa900ba034cd52) in files app/components/Vault/index.js. This address is from token `CRV`  
- Found address [0xc5bddf9843308380375a611c18b50fb9341f502a](https://etherscan.io/address/0xc5bddf9843308380375a611c18b50fb9341f502a) in files app/components/Vault/index.js. This address is from vault `yveCRV`

**Review by xgambitox — APPROVED** [review]

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found address [0xd533a949740bb3306d119cc777fa900ba034cd52](https://etherscan.io/address/0xd533a949740bb3306d119cc777fa900ba034cd52) in files app/components/Vault/index.js. This address is from token `CRV`  
- Found address [0xc5bddf9843308380375a611c18b50fb9341f502a](https://etherscan.io/address/0xc5bddf9843308380375a611c18b50fb9341f502a) in files app/components/Vault/index.js. This address is from vault `yveCRV`  
- Found address [0x2a38B9B0201Ca39B17B460eD2f11e4929559071E](https://etherscan.io/address/0x2a38B9B0201Ca39B17B460eD2f11e4929559071E) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-GUSD`  
- Found address [0x4B5BfD52124784745c1071dcB244C6688d2533d3](https://etherscan.io/address/0x4B5BfD52124784745c1071dcB244C6688d2533d3) in files app/containers/Vaults/constants.js. This address is from vault `yUSD`  
- Found address [0x8ee57c05741aA9DB947A744E713C15d4d19D8822](https://etherscan.io/address/0x8ee57c05741aA9DB947A744E713C15d4d19D8822) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-yBUSD`  
- Found address [0xD6Ea40597Be05c201845c0bFd2e96A60bACde267](https://etherscan.io/address/0xD6Ea40597Be05c201845c0bFd2e96A60bACde267) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-Compound`  
- Found address [0x84E13785B5a27879921D6F685f041421C7F482dA](https://etherscan.io/address/0x84E13785B5a27879921D6F685f041421C7F482dA) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-3pool`  
- Found address [0x6Ede7F19df5df6EF23bD5B9CeDb651580Bdf56Ca](https://etherscan.io/address/0x6Ede7F19df5df6EF23bD5B9CeDb651580Bdf56Ca) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-BUSD`  
- Found address [0xC4dAf3b5e2A9e93861c3FBDd25f1e943B8D87417](https://etherscan.io/address/0xC4dAf3b5e2A9e93861c3FBDd25f1e943B8D87417) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-USDP`  
- Found address [0x5fA5B62c8AF877CB37031e0a3B2f34A78e3C56A6](https://etherscan.io/address/0x5fA5B62c8AF877CB37031e0a3B2f34A78e3C56A6) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-LUSD`  
- Found address [0x3B96d491f067912D18563d56858Ba7d6EC67a6fa](https://etherscan.io/address/0x3B96d491f067912D18563d56858Ba7d6EC67a6fa) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-USDN`

File: app/containers/Vaults/constants.js [hunk 1]
```diff
@@ -56,11 +56,10 @@ export const AMPLIFY_VAULTS_ADDRESSES = [
...
 export const LAZY_APE_ADDRESSES = [
-  '0x2a38B9B0201Ca39B17B460eD2f11e4929559071E',
-  '0x4B5BfD52124784745c1071dcB244C6688d2533d3',
-  '0x8ee57c05741aA9DB947A744E713C15d4d19D8822',
-  '0xD6Ea40597Be05c201845c0bFd2e96A60bACde267',
-  '0x84E13785B5a27879921D6F685f041421C7F482dA',
+  '0x6Ede7F19df5df6EF23bD5B9CeDb651580Bdf56Ca',
+  '0xC4dAf3b5e2A9e93861c3FBDd25f1e943B8D87417',
+  '0x5fA5B62c8AF877CB37031e0a3B2f34A78e3C56A6',
+  '0x3B96d491f067912D18563d56858Ba7d6EC67a6fa ',
```
Comment (xgambitox) [inline app/containers/Vaults/constants.js:62]: There is a space at the end of the string, so button does not displays for crvUSDN

**Review by xgambitox — CHANGES_REQUESTED** [review]

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found address [0xd533a949740bb3306d119cc777fa900ba034cd52](https://etherscan.io/address/0xd533a949740bb3306d119cc777fa900ba034cd52) in files app/components/Vault/index.js. This address is from token `CRV`  
- Found address [0xc5bddf9843308380375a611c18b50fb9341f502a](https://etherscan.io/address/0xc5bddf9843308380375a611c18b50fb9341f502a) in files app/components/Vault/index.js. This address is from vault `yveCRV`  
- Found address [0x2a38B9B0201Ca39B17B460eD2f11e4929559071E](https://etherscan.io/address/0x2a38B9B0201Ca39B17B460eD2f11e4929559071E) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-GUSD`  
- Found address [0x4B5BfD52124784745c1071dcB244C6688d2533d3](https://etherscan.io/address/0x4B5BfD52124784745c1071dcB244C6688d2533d3) in files app/containers/Vaults/constants.js. This address is from vault `yUSD`  
- Found address [0x8ee57c05741aA9DB947A744E713C15d4d19D8822](https://etherscan.io/address/0x8ee57c05741aA9DB947A744E713C15d4d19D8822) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-yBUSD`  
- Found address [0xD6Ea40597Be05c201845c0bFd2e96A60bACde267](https://etherscan.io/address/0xD6Ea40597Be05c201845c0bFd2e96A60bACde267) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-Compound`  
- Found address [0x84E13785B5a27879921D6F685f041421C7F482dA](https://etherscan.io/address/0x84E13785B5a27879921D6F685f041421C7F482dA) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-3pool`  
- Found address [0x6Ede7F19df5df6EF23bD5B9CeDb651580Bdf56Ca](https://etherscan.io/address/0x6Ede7F19df5df6EF23bD5B9CeDb651580Bdf56Ca) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-BUSD`  
- Found address [0xC4dAf3b5e2A9e93861c3FBDd25f1e943B8D87417](https://etherscan.io/address/0xC4dAf3b5e2A9e93861c3FBDd25f1e943B8D87417) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-USDP`  
- Found address [0x5fA5B62c8AF877CB37031e0a3B2f34A78e3C56A6](https://etherscan.io/address/0x5fA5B62c8AF877CB37031e0a3B2f34A78e3C56A6) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-LUSD`  
- Found address [0x3B96d491f067912D18563d56858Ba7d6EC67a6fa](https://etherscan.io/address/0x3B96d491f067912D18563d56858Ba7d6EC67a6fa) in files app/containers/Vaults/constants.js. This address is from vault `yvCurve-USDN`

**Comment by patcito** [comment]
pushed a fix

**Review by xgambitox — APPROVED** [review]

**Review by dudesahn — APPROVED** [review]

**Pull request merged**